### DISCUSSION TITLE
Cannot save Edit form with relationship data

### DISCUSSION THREAD

**Discussion by happyclicker** [filamentphp/filament#10703] [Help]
(v3) I'm doing something that's pretty simple - I have a resource of Items.  I have build a form, and one of the Select fields is a relationship that I want to display informational data from (in this case, the manufacturer name).  I don't want this field updated when saving this form, I just want to display it.  The create process for an item is done by getting data from another system, and only certain fields are editable in this panel (the other system is the "master" for most fields).

`Select::make('manufacturer_id')
                    ->label('Manufacturer')
                    ->disabled()
                    ->relationship('manufacturers', 'man')->dehydrated(false),`

What happens when I hit "Save Changes", I get this error during the validation process:

**Call to undefined method Illuminate\Database\Eloquent\Relations\HasOne::associate()**

It is my understanding that calling the Select in this way, the manufacturer_id filed shouldn't be passed back to the server for validation from the docs, but it clearly is.  I have commented out this Select::make code and the save works with no issues.  

Do I have to create a custom form instead to display this type of data?   Or is there a way to create a spot in the form that is just a piece of text?  I feel like I am missing something basic.

**Comment by leandrocfe** [comment]
what about `->dehydrated(true)`?
This shouldn't auto save the relationship..

**Reply by happyclicker** [reply]
It makes no difference.  

I did figure out that this happens after the beforeValidate() call, so it appears it is related to the validation logic.
